### ISSUE TITLE
Webhook Publish, and Unpublish events not showing in UI

### ISSUE THREAD

**Issue by jlmelis** [strapi/strapi#15518]
### Required System information

- Node.js version: v16.17.0
- NPM version: 8.192
- Strapi version: v4.5.6
- Database: Postgres
- Operating system: Windows

### Describe the bug

When creating a web hook the "Publish" and "Unpublish" aren't showing up in the UI. If I manually add:

[  "entry.publish",
  "entry.unpublish"]

in the events column of the strapi_webhooks table the webhooks triggers as expected on publish and unpublish. The only events available in the UI are "Create", "Update", and "Delete". Should "Publish" and "Unpublish" be available in the UI? Is there a setting or something I am missing to get them to show in the UI?

### Expected behavior

Check boxes for Publish and Unpublish would show in the events section of a webhook.

### Screenshots

![image](https://user-images.githubusercontent.com/11036707/213791563-2c2089ad-373c-4042-b195-67177f3bf751.png)

![image](https://user-images.githubusercontent.com/11036707/213791898-c7be5b17-a7a8-412f-965e-38e291429641.png)

**Comment by dogusdeniz** [comment]
I think can't be possible because publish and unpublished triggered update

**Comment by jlmelis** [comment]
> I think can't be possible because publish and unpublished triggered update

I'm not sure I understand. When I tried to use the "update" event the webhook fires on save (i.e. saving an unpublished draft) and publish. When I manually add the "entry.pulish" and "entry.unpublish" events directly to the database the we hook only fires on those events. It doesn't not fire when I save an unpublished draft. So the events seem to work as expected there is just no way that I can see to set them in the UI.

**Comment by jlmelis** [comment]
Sorry closed by accident.

**Comment by dogusdeniz** [comment]
If I publish it, the published_at date is set in the database and this is trigger the update event. If I unpublish it, the published_at is set null and trigger the update event.

**Comment by dubiao** [comment]
> If I publish it, the published_at date is set in the database and this is trigger the update event. If I unpublish it, the published_at is set null and trigger the update event.

Yes. But I think @jlmelis hopes that there will be a UI component (checkbox or selector) stands for publish/unpublish event, but not only the fields the class defined.

**Comment by pravind-hinge** [comment]
Any updates on this issue ?
